CONTRIBUTING.md

```diff
@@ -0,0 +1,72 @@
+
+The community owned and driven Presto project is supported by the [Presto Foundation](https://prestodb.io), an independent nonprofit organization with open and neutral governance, hosted under the [Linux Foundation](https://linuxfoundation.org). Learn more about the [Presto Foundation](https://prestodb.io) here.
+
+All Presto development occurs on [GitHub](https://github.com/prestodb), following open source best practices.
+
+## Technical governance
+
+Presto follows industry best practices, with the technical project and its governance distinctly separate from the Presto Foundation. The [Technical Charter](https://github.com/prestodb/tsc/blob/master/CHARTER.md) establishes the Presto Technical Steering Committee as the body which oversees all technical aspects of the open source project. By contrast, the [Presto Foundation](https://prestodb.io) Governing Board is responsible for raising and distributing funds to support project activities, and setting policy. While the two groups are separate, they work in close coordination to establish, guide, and support activities that lead to a long-term, sustainable future for Presto.
+
+### Contributors
+
+Contributors are any members of the open source community who participate in project develompent. Contributors may participate on their own behalf or on behalf of their employer. Membership in the [Presto Foundation](https://prestodb.io) is not required in order to contribute, although contributors must have signed a CLA prior to their code being merged (see below).
+
+### Committers
+
+Committers are members of the contributor community who have demonstrated an ongoing commitment to the health and sustainability of the Presto project through good citizenship and wise stewardship. These individuals have been elevated to a decision-making role, and responsible for reviewing and merging code from contributors. They may also be involved in the release process or other critical project roles.
+
+Current Presto committers:
+
+| Name              | Github                                            | Affiliation |
+| ----------------- | ------------------------------------------------- | ------------|
+| Masha Basmanova   |[mbasmanova](https://github.com/mbasmanova)        | Facebook    |
+| Zhenxiao Luo      |[zhenxiao](https://github.com/zhenxiao)            | Twitter     |
+| Andrii Rosa       |[arhimondr](https://github.com/arhimondr)          | Facebook    |
+| Rebecca Schlussel |[rschlussel](https://.github.com/rschlussel)       | Facebook    |
+| James Sun         |[highker](https://github.com/highker)              | Facebook    |
+| Wenlei Xie        |[wenleix](https://github.com/wenleix)              | Facebook    |
+| Rongrong Zhong    |[rongrong](https://github.com/rongrong)            | Facebook    |
+| Shixuan Fan       |[shixuan-fan](https://github.com/shixuan-fan)      | Facebook    |
+| Jiexi Lin         |[jessesleeping](https://github.com/jessesleeping)  | Facebook    |
+
```

Comment:
We recently just added a new committer:
``` 
| Leiqing Cai  | [caithagoras](https://github.com/caithagoras) | Facebook |
```
Could you please add him in. Thanks!

Reply:
Thanks @rongrong, will do!  I'll push a patch now for that, and also adding Venki.